### PR TITLE
fix(connectors): restructuring project so connectors pass important props

### PR DESCRIPTION
**Pull request by anthony-liddle**
## Goal

The main logic for all components should live in connectors. App.js now only manages open state

### CODE REVIEW

File: src/components/tagging/taginput/taginput.js [hunk 1]
```diff
@@ -164,7 +163,6 @@ TagInput.propTypes = {
   handleRemoveAction: PropTypes.func,
   makeTagsInactive: PropTypes.func,
   setValue: PropTypes.func,
-  // inputRef: PropTypes.func,
```
Comment (anthony-liddle) [inline src/components/tagging/taginput/taginput.js:167]: Just some cleanup

File: src/components/doorhanger/doorhanger.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+export const Doorhanger = ({ isOpen, children }) => {
+  return (
+    <div className={doorhangerStyle}>
+      <div className={cx('doorHanger', isOpen && 'open')}>
+        { children }
+      </div>
+    </div>
+  )
+}
```
Comment (anthony-liddle) [inline src/components/doorhanger/doorhanger.js:43]: Doorhanger is now a simple component that is either open or closed! That's it 🤷

File: src/connectors/footer/footer.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Footer } from 'components/footer/footer'
+
+export const FooterConnector = () => {
+  const settingsClick = () => console.log('settings clicked')
```
Comment (anthony-liddle) [inline src/connectors/footer/footer.js:5]: This will be the function to open the Options page

**Review by wtfluckey — APPROVED** [review]
I like these being broken out into their own connectors. Good stuff.

**Pull request merged**